manifests/agent.pp

```diff
@@ -0,0 +1,18 @@
+#
+# == Class: pulp::agent
+#
+# Install and configure Pulp agent
+#
+class pulp::agent ($version) {
```

Comment:
Is this supposed to be able to support a stand-alone agent on a consumer? Looking below at the checks on `pulp::config` and `server.conf` it would seem this wouldn't work on a pure consumer.